src/app/navigation/navigation.component.html

```diff
@@ -3,12 +3,12 @@
     <mat-icon class="logo" svgIcon="logo"></mat-icon>
   </a>
   <div class="left-buttons">
-    <button mat-button routerLink="/program" routerLinkActive="active-link">
-      Program
-    </button>
-    <button mat-button routerLink="/voting" routerLinkActive="active-link">
-      Voting
-    </button>
+    <a routerLink="/program" class="button-text-color">
+      <button mat-button>Program</button>
+    </a>
+    <a routerLink="/voting" class="button-text-color">
+      <button mat-button>Voting</button>
+    </a>
```

Comment:
```suggestion
    <button mat-button routerLink="/program" routerLinkActive="active-link">
      Program
    </button>
    <button mat-button routerLink="/voting" routerLinkActive="active-link">
      Voting
    </button>
```

I think these changes should be reverted since the navbar works fine on `develop`. Here, the active page is not highlighted.